src/pkg/packaging/dir.proj

```diff
@@ -13,9 +13,10 @@
     </PackageTargets>
   </PropertyGroup>
 
-  <Target Name="Build" DependsOnTargets="BuildInstallers;BuildCombinedInstallers" Condition="'$(DOTNET_BUILD_SKIP_PACKAGING)' != 'true'" />
+  <Target Name="Build" DependsOnTargets="BuildInstallers;BuildCombinedInstallers;BuildVSNugetPackages" Condition="'$(DOTNET_BUILD_SKIP_PACKAGING)' != 'true'" />
   <Target Name="BuildInstallers" DependsOnTargets="$(PackageTargets)" Condition="'$(DOTNET_BUILD_SKIP_PACKAGING)' != 'true'" />
   <Target Name="BuildCombinedInstallers" DependsOnTargets="GenerateCombinedInstallers" Condition="'$(DOTNET_BUILD_SKIP_PACKAGING)' != 'true'" />
+  <Target Name="BuildVSNugetPackages" DependsOnTargets="GenerateVSNugetPackages" Condition="'$(DOTNET_BUILD_SKIP_PACKAGING)' != 'true'" />
```

Comment:
Sorry, forgot the reason these targets are segmented this way--the `BuildInstallers` and `BuildCombinedInstallers` targets are called individually, so the mock build (https://dev.azure.com/dnceng/internal/_build/results?buildId=236815&view=logs) missed your target entirely. I think your best bet is to just stick `GenerateVSNugetPackages` in `DependsOnTargets="GenerateCombinedInstallers"`.

https://github.com/dotnet/core-setup/blob/e1d780539e85f4d8de263957715f9d08db2ceef4/eng/jobs/windows-build.yml#L151-L152

Reply:
Done.